src/main/java/de/hysky/skyblocker/config/categories/HelperCategory.java

```diff
@@ -209,6 +209,14 @@ public static ConfigCategory create(SkyblockerConfig defaults, SkyblockerConfig
 										rarity -> config.helpers.fishing.minimumNotificationRarity = rarity == SkyblockItemRarity.DIVINE ? SkyblockItemRarity.UNKNOWN : rarity)
 								.controller(ConfigUtils::createEnumCyclingListController)
 								.build())
+						.option(Option.<Boolean>createBuilder()
+								.name(Text.translatable("skyblocker.config.helpers.fishing.enableFishingHookDisplay"))
+								.description(OptionDescription.of(Text.translatable("skyblocker.config.helpers.fishing.enableFishingHookDisplay.@Tooltip")))
+								.binding(defaults.helpers.fishing.enableFishingHookDisplay,
+										() -> config.helpers.fishing.enableFishingHookDisplay,
+										newValue -> config.helpers.fishing.enableFishingHookDisplay = newValue)
+								.controller(ConfigUtils::createBooleanController)
+								.build())
```

Comment:
This is not in the same place as the config class. I would move this above to above `enableFishingTimer` and below  `enableFishingHelper`.